tests/Console/Command/DescribeCommandTest.php

```diff
@@ -57,21 +56,64 @@ public function testExecuteOutput()
    @@ @@
    -<?php echo 'bad stuff and bad thing';
    +<?php echo 'good stuff and good thing';
-%spaces%
    ----------- end diff -----------
 
 
 EOT;
 
         $this->assertSame(
-            str_replace(array('%spaces%', "\n"), array('   ', PHP_EOL), $expected),
-            $this->execute()->getDisplay()
+            str_replace(array("\n"), array(PHP_EOL), $expected),
+            $this->execute(false)->getDisplay()
+        );
+    }
+
+    public function testExecuteOutputWithDecoration()
+    {
+        $expected = <<<EOT
+\033[32mDescription of\033[39m Foo/bar \033[32mrule\033[39m.
+Fixes stuff.
+Replaces bad stuff with good stuff.
+
+Fixer is configurable using following options:
+* \033[32mthings\033[39m (\033[33mbool\033[39m): enables fixing things as well; defaults to \033[33mfalse\033[39m
+
+Fixing examples:
+ * Example #1.
+\033[33m   ---------- begin diff ----------\033[39m
+   \033[31m--- Original\033[39m
+   \033[32m+++ New\033[39m
+   \033[36m@@ @@\033[39m
+   \033[31m-<?php echo 'bad stuff and bad thing';\033[39m
+   \033[32m+<?php echo 'good stuff and bad thing';\033[39m
+\033[33m   ----------- end diff -----------\033[39m
+
+ * Example #2. Fixing with configuration: \033[33m['things' => true]\033[39m.
+\033[33m   ---------- begin diff ----------\033[39m
+   \033[31m--- Original\033[39m
+   \033[32m+++ New\033[39m
+   \033[36m@@ @@\033[39m
+   \033[31m-<?php echo 'bad stuff and bad thing';\033[39m
+   \033[32m+<?php echo 'good stuff and good thing';\033[39m
+\033[33m   ----------- end diff -----------\033[39m
+
+
+EOT;
+
+        $actual = $this->execute(true)->getDisplay();
+
+        if (false !== strpos($actual, "\033[0m")) {
+            $expected = str_replace("\033[39m", "\033[0m", $expected);
```

Comment:
This can be removed when merging up to 2.3 as this does not occur anymore with `symfony/console:^3.0`.